tests/plugins/balloontoolbar/manual/context-api.md

```diff
@@ -0,0 +1,11 @@
+@bender-ui: collapsed
+@bender-tags: 4.8.0, feature, 1346, balloontoolbar
+@bender-ckeditor-plugins: wysiwygarea,toolbar,undo,basicstyles,notification,floatingspace,sourcearea,list,link
```

Comment:
When creating tests, remember to include plugins which are only necessary for the tests to pass/fail. In this case, you could just skip all plugins as there is not user interaction with the editor.
However, it is better to make editor properly load (so it doesn't looks like test is broken) so you could use `wysiwygarea,toolbar,basicstyles`.